examples/compose/docker-compose-with-web.yml

```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 512m
+    restart: always
+    environment:
+      - CONSUL=consul
+    links:
+      - consul:consul
```

Comment:
The link pattern is easer for demos, but it's being deprecated by Docker itself. Should you add a comment giving a suggestion for how to do it in production?